en/pocket/firefox/new-tab.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://dev.tekcopteg.com/firefox/new_tab_learn_more
+
+pocket-new-tab-learn-more = Learn More
+pocket-new-tab-firefox = { -brand-name-firefox }
+pocket-new-tab-pocket = { -brand-name-pocket }
+pocket-new-tab-pocket-is-part = { -brand-name-pocket } is part of the { -brand-name-firefox } family
+
+pocket-new-tab-pocket-for-firefox = { -brand-name-pocket } for { -brand-name-firefox }
+pocket-new-tab-build-your-personal = Build your personal library of fascinating reads.
+# Variables:
+# $pocket-logo-inline (string) class for Pocket’s inline logo
+pocket-new-tab-included-inside = Included inside { -brand-name-firefox }, the <span class="{ $pocket-logo-inline }">{ -brand-name-pocket } Logo</span> button lets you save articles from across the web and read them in a quiet, private space.
+pocket-new-tab-activate-pocket = Activate { -brand-name-pocket } in { -brand-name-firefox }
+pocket-new-tab-more-ways-to = More ways to sign up
+
+pocket-new-tab-discover-the-most = Discover the most thought-provoking stories out there, curated by { -brand-name-pocket }.
+# Variables
+#   learn_url (url) - link to https://help.getpocket.com/article/1142-firefox-new-tab-recommendations-faq#why
```

Comment:
This comment should be found to string ID `pocket-new-tab-as-part-of`, the one after the string below.